src/main/java/teammates/storage/sqlentity/FeedbackSession.java

```diff
@@ -112,6 +112,35 @@ public FeedbackSession(String name, Course course, String creatorEmail, String i
         this.setPublishedEmailEnabled(isPublishedEmailEnabled);
     }
 
+    /**
+     * Creates a copy that uses the specific deadline for the given student.
+     *
+     * @param studentEmail The email address of the given student.
+     * @return The copy of this object for the given student.
+     */
+    public FeedbackSession getCopyForStudent(String studentEmail) {
+        FeedbackSession copy = getCopy();
+        for (DeadlineExtension de : copy.getDeadlineExtensions()) {
+            if (!de.getUser().getEmail().equals(studentEmail)) {
+                de.setEndTime(copy.getEndTime());
+            }
+        }
+        return copy;
+    }
```

Comment:
We are not using this getCopyForStudent and getCopyForInstructor anymore, doubt this is necessary.

Reply:
I think we are still using `getCopyForStudent` in GetFeedbackSessionsAction line 100